script/release

```diff
@@ -62,13 +54,14 @@ puts
 puts "From: #{original_version}"
 puts "To:   #{version_string}"
 puts
+
 puts "Type 'yes' to continue"
 x = STDIN.gets
 exit(1) unless x.strip == 'yes'
 puts
 
 verbose('git fetch origin')
-verbose('git reset --hard origin/master')
+verbose('git reset --hard origin/master') if release_option != "5"
```

Comment:
should we do reset --hard origin/hotfix?

Reply:
we do not fix the branch name for hotfix so this would not be possible.